code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -49,7 +49,7 @@
 
 /obj/item/weapon/pinpointer/examine(mob/user)
 	..(user)
-	for(var/obj/machinery/nuclearbomb/bomb in world)
+	for(var/obj/machinery/nuclearbomb/bomb in machines)

```

Comment:
If one is, the actual processing machine list(s) should be moved into SSmachinery.